msteams-platform/toolkit/configure-message-extension-capability.md

```diff
@@ -0,0 +1,632 @@
+
+```json
+"composeExtensions": [
+    {
+        "botId": "${{BOT_ID}}",
+        "commands": [
+            {
+                "id": "createCard",
+                "context": [
+                    "compose"
+                ],
+                "description": "Command to run action to create a Card from Compose Box",
+                "title": "Create Card",
+                "type": "action",
+                "parameters": [
+                    {
+                        "name": "title",
+                        "title": "Card title",
+                        "description": "Title for the card",
+                        "inputType": "text"
+                    },
+                    {
+                        "name": "subTitle",
+                        "title": "Subtitle",
+                        "description": "Subtitle for the card",
+                        "inputType": "text"
+                    },
+                    {
+                        "name": "text",
+                        "title": "Text",
+                        "description": "Text for the card",
+                        "inputType": "textarea"
+                    }
+                ]
+            },
+            {
+                "id": "shareMessage",
+                "context": [
+                    "message"
+                ],
+                "description": "Test command to run action on message context (message sharing)",
+                "title": "Share Message",
+                "type": "action",
+                "parameters": [
+                    {
+                        "name": "includeImage",
+                        "title": "Include Image",
+                        "description": "Include image in Hero Card",
+                        "inputType": "toggle"
+                    }
+                ]
+            },
+            {
+                "id": "searchQuery",
+                "context": [
+                    "compose",
+                    "commandBox"
+               ],
+               "description": "Test command to run query",
+               "title": "Search",
+                "type": "query",
+                "parameters": [
+                    {
+                        "name": "searchQuery",
+                        "title": "Search Query",
+                        "description": "Your search query",
+                        "inputType": "text"
+                    }
+                ]
+            }
+        ],
+        "messageHandlers": [
+            {
+                "type": "link",
+                "value": {
+                    "domains": [
+                        "*.botframework.com"
+                    ]
+                }
+            }
+        ]
+    }
+]
+```
+
+### Bring message extension code to your project
```

Comment:
Add message extension code to your project

Reply:
done